ecs-init/docker/docker.go

```diff
@@ -273,3 +278,18 @@ func (c *Client) StopAgent() error {
 	stopContainerTimeoutSeconds := uint(10)
 	return c.docker.StopContainer(id, stopContainerTimeoutSeconds)
 }
+
+// getPlatformSpecificEnvVariables gets a map of environment variable key-value
```

Comment:
Can you move this back?  I believe this was intended for the task networking changes, which will only be enabled on Amazon Linux right now.

Reply:
I had moved it because it was literally the same on both platforms. Is this changing?

Reply:
@aaithal would know better than me.  I think that's why it's the way it is.

Reply:
It's intended to be changed in the next init release. Please move it back.